src/metatensor/models/cli/conf/architecture/alchemical_model.yaml

```diff
@@ -0,0 +1,25 @@
+# default hyperparameters for the Alchemical Model
+name: alchemical_model
+
+model:
+  soap:
+    num_pseudo_species: 4
+    basis_normalization_factor: 1.0
+    cutoff_radius: 5.0
+    basis_cutoff: 400
+    radial_basis_type: 'physical'
+    basis_scale: 3.0
+    trainable_basis: true
+
+  bpnn:
+    num_hidden_layers: 2
+    num_neurons_per_layer: 32
+    activation_function: SiLU
+
+training:
+  batch_size: 8
+  num_epochs: 100
+  learning_rate: 0.001
+  log_interval: 10
+  checkpoint_interval: 25
+  device: 'cpu'
```

Comment:
This should not be here but will be passed directly from the train function.

Reply:
These lines are somehow needed for training step in `test_regression.py`

Reply:
Yes these lines but the device is a global parameter and will be provided via the train function.